services/gundeck/test/unit/Push.hs

```diff
@@ -34,7 +33,8 @@ import Test.Tasty.QuickCheck
 
 tests :: TestTree
 tests =
-  testGroup "bulkpush" $
+  testGroup
+    "bulkpush"
```

Comment:
Same here.  (The use of `$` here is to keep the first two arguments in the same line.)